### ISSUE TITLE
Indicate failed recipes

### ISSUE THREAD

**Issue by mikenholt** [ngageoint/scale-ui#141]
On the 'Recipes' page it should be more obvious that a recipe is done processing when it fails.  I purpose putting 'Failed' in the 'Completed' column.  
![Screenshot 2019-06-22 19 59 52](https://user-images.githubusercontent.com/22593495/59969931-5fccd280-9528-11e9-986d-fc7c8bc056d2.png)

**Comment by mikenholt** [comment]
Let us know if you need additional info from the API to support this.

**Comment by ericsvendsen-mil** [comment]
Yes, I need additional info from the API for this. All I get right now is a `completed` field, which apparently only populates when a recipe completes successfully? There's nothing in the recipes list payload that would indicate whether one has failed or not.

**Comment by ericsvendsen-mil** [comment]
I'm going to move this to the scale repo for now. Feel free to move it back once there's something in place for this.
